lua/neogit/status.lua

```diff
@@ -180,15 +180,16 @@ local function draw_buffer()
   if git.repo.head.branch == "(detached)" then
     output:append(
       string.format(
-        "Head:     (detached) at %s %s",
```

Comment:
Yes.

The teo reasons I decided to try this way is so that the commit hashes line up with Merge and attached Head states,

and the second is that vim didn't quite seem to want to match or escape the parenthesis inside the `syn region` 😅